tests/DS/sds_detect_version/test_detect_version.sh

```diff
@@ -16,12 +16,21 @@ function test_oscap_info {
 	version="$1"
 	stdout="$(mktemp)"
 	stderr="$(mktemp)"
-	$OSCAP info $srcdir/scap-$version-ds.xml > $stdout 2> $stderr
+	ds="$(mktemp)"
+	cp scap-ds.xml $ds
+	sed -i "s/X.X/${version}/g" $ds
+
+	$OSCAP info $ds > $stdout 2> $stderr
 	[ ! -s $stderr ]
 	grep -q "Version: $version" $stdout
 	rm $stdout
 	rm $stderr
+	rm $ds
 }
 
-test_oscap_info "1.2"
-test_oscap_info "1.3"
+SDS=$(find $builddir/schemas/sds -maxdepth 1 -mindepth 1 -type d -printf '%f\n')
```

Comment:
I think it should be `$top_srcdir` instead of `$builddir`.

Reply:
I needed to add this variable to the `Makefile.am`, this was the only way to make it work. Is it correct?

Reply:
It seems the right way. But please don't rename it, keep `$top_srcdir` to be consistent across the whole project.

Reply:
Done